website/content/docs/upgrade/upgrade-specific.mdx

```diff
@@ -51,6 +51,26 @@ The volume staging directory for new CSI plugin tasks will now be
 mounted to the task's `NOMAD_TASK_DIR` instead of the
 `csi_plugin.mount_config`.
 
+#### Server Raft Database
```

Comment:
We should mention the client as well... this is a bit wordy but would work:

```suggestion
#### Server Raft and Client State Databases
```

Content could go _after_ the server and be something like:

> Likewise Nomad client's store some local state in a bolt database that will be upgraded to the new implementation. Downgrading clients in place is not possible. The client must be drained and have its state directory removed before restarting with an old version.

Reply:
Just a heads up that we already have a note about client metadata upgrades in 1.3.0 because of https://github.com/hashicorp/nomad/pull/12078

Reply:
Yup, Tim's note already covers the client case

https://github.com/hashicorp/nomad/pull/12078/files#diff-7c21f2d55cea6dc7286b54d164df00955882370ed3dbdeccd055f89a48350cf2R28

Reply:
Fantastic! Sorry for the noise.